client/packages/invoices/src/Returns/api/api.ts

```diff
@@ -340,7 +340,7 @@ export const getReturnsQueries = (sdk: Sdk, storeId: string) => ({
     const { insertCustomerReturn } = result;
 
     if (insertCustomerReturn.__typename === 'InvoiceNode') {
-      return insertCustomerReturn.invoiceNumber;
+      return insertCustomerReturn;
```

Comment:
returning the full return so that navigation to return from AppBarButtons still works with invoice number